feature-libs/quote/components/details/edit/quote-details-edit.component.html

```diff
@@ -0,0 +1,67 @@
+<div *ngIf="content" class="cx-card">
+  <!-- Card Body -->
+  <div class="card-body cx-card-body cx-card-edit">
+    <!-- Card title -->
+    <span class="cx-card-title">
+      {{ 'quote.details.information' | cxTranslate }}
+    </span>
+    <!-- Card Content -->
+    <div class="cx-card-container">
+      <!-- Card Label -->
+      <div class="cx-card-label-container">
+        <form [formGroup]="editForm">
+          <div class="cx-card-paragraph">
+            <div class="cx-card-paragraph-title">
+              {{ 'quote.details.name' | cxTranslate }}
+            </div>
+            <div class="form-group">
+              <input
+                class="form-control"
+                type="text"
+                name="name"
+                formControlName="name"
+              />
+            </div>
+          </div>
+          <div class="cx-card-paragraph">
+            <div class="cx-card-paragraph-title">
+              {{ 'quote.details.description' | cxTranslate }}
+            </div>
+            <div class="form-group">
+              <textarea
+                class="form-control"
+                [maxLength]="content.charactersLimit"
+                rows="4"
+                name="description"
+                formControlName="description"
+              ></textarea>
+              <ng-container *ngIf="content.charactersLimit">
+                <p class="cx-info-text">
```

Comment:
you could also place the *ngIf on the p element to get rid of the enclosing ng-container

Reply:
Agree! Done!